meta-ibm/recipes-phosphor/vpd/ibm-vpd-parser/50001001.json

```diff
@@ -223,6 +223,17 @@
         "extraInterfaces": {
           "com.ibm.ipzvpd.Location" : {
               "LocationCode" : "Ufcs-A0"
+          },
+          "com.ibm.ipzvpd.VINI" : {
+              "FN": [48, 50, 89, 75, 51, 50, 55],
+              "CC": [55, 66, 53, 71]
+          },
+          "com.ibm.ipzvpd.DINF" : {
+              "RI": [0, 5, 33, 0]
+          },
+          "xyz.openbmc_project.Inventory.Decorator.Asset" : {
+              "Model": "7B5G",
+              "SparePartNumber": "02YK327"
```

Comment:
in issue 1592 I wrote down ...237 instead of 327.  Was that just a typo on my part?

Reply:
I definitely read that wrong. If you can confirm it is 237, I can make another PR to get that fixed, since this one is merged already

Reply:
I doublechecked what I got from Nicole, and it is 02YK237

Reply:
Thanks, Matt. Fixed in: https://github.com/ibm-openbmc/openbmc/pull/176. Please review.